CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (x/gov) [#18707](https://github.com/cosmos/cosmos-sdk/pull/18707) Improve genesis validation. 
```

Comment:
Adding the changelog here, even though it should be in x/gox/changelog.md, as we backport this, we'll remove the line anyway at next point version.